packages/manager/apps/zimbra/src/components/layout-helpers/Dashboard/Dashboard.tsx

```diff
@@ -40,22 +47,34 @@ export const Dashboard: React.FC<DashboardLayoutProps> = () => {
       name: 'general_informations',
       title: t('zimbra_dashboard_general_informations'),
       to: basePath,
+      pathMatchers: [urls.dashboard].map(
+        (path) => new RegExp(path.replace(':serviceName', platformId)),
+      ),
```

Comment:
Should be move in a dedicated function to avoid duplicating code

```
function computePathMatchers (routes) : RegExp[] {
  return routes.map(
        (path) => new RegExp(path.replace(':serviceName', platformId)),
      )
}
```